src/components/Timelog/Timelog.jsx

```diff
@@ -530,7 +530,7 @@ const Timelog = props => {
                           </div>
                         </div>
                       ) : (
-                        canPutUserProfileImportantInfo && (
+                        (canPutUserProfileImportantInfo || canAddTimeEntry) && (
```

Comment:
Should `canAddTimeEntry` be replacing `canPutUserProfileImportantInfo` entirely or just added as it is now?

Reply:
We can just replace it entirely.